docs/contributing/gatsby-governance-model.md

```diff
@@ -35,7 +35,7 @@ We accept all types of contributions to [`packages`](https://github.com/gatsbyjs
 - [`gatsby`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby)
```

Comment:
Rather than " with these exceptions:", how about something like "but the following packages have extra requirements:"